docs/design/job_architecture.md

```diff
@@ -278,133 +175,68 @@ These are all caught by the `JobCommChannel` on the browser side, then parsed an
+By design, these messages are only seen by the `JobCommChannel`; the JobComm interface does not constitute an API for narrative job data.
 
 ### `ERROR`
-A general job comm error, capturing most errors that get thrown by the kernel
 
-**content** (this varies, but usually includes the below)
-  * `raw_request` - the original request message that wound up in an error
-  * `source` - request type or method that triggered the error
+A general job comm error message, capturing exceptions that get thrown by the kernel. The frontend uses the `request` data to distribute error messages on the appropriate job or cell channels.
+
+Content varies, but usually includes the below:
+  * `source` - what triggered the error; this may be the request type, arbitrary string, or null, meant to indicate what triggered the error
+  * `request` - request data, arbitrary string, or null, meant to indicate the comm request that triggered the error.
+
 Error messages usually also contain data about the error itself:
-  * `name` - the error type
-  * `message` - description of the issue
+  * `name` - the exception type name
+  * `message` - the exception message
 
-**bus** `ERROR`
 
-### `INFO`
-Includes information about the running job
+### `INFO` (response)
+
+Includes information about the running job. The frontend splits out messages and distributes them by job or batch ID.
 
-**content**
-Dictionary with key(s) job ID and value dictionaries with the following structure:
+Job metadata, keyed by job ID, with the following structure:
   * `app_id` - string, the app id (format = `module_name/app_name`)
   * `app_name` - string, the human-readable app name
   * `job_id` - string, the job id
-  * `job_params` - the unstructured set of parameters sent to the execution engine
+  * `job_params` - array of job parameters sent to the execution engine
   * `batch_id` - id of batch container job
 
-i.e.
-```json
-{
-  "job_id_1": {
-    "app_id": ...,
-    "app_name": ...,
-    "job_id": "job_id_1",
-    "job_params": ...,
-    "batch_id": "some_batch_id",
-  },
-  "job_id_2": { ...contents... }
-}
-```
-
-In case of error, the response has the keys:
+In case of error, the response has instead the keys:
   * `job_id`
   * `error` - brief message explaining the issue
 
-**bus**
-The frontend splits out the job info and distributes it out by job ID.
-
-### `STATUS`
-The current job state. This one is probably most common.
-
-**content**
-Dictionary with key(s) job ID and value dictionaries with the following structure:
-  * `jobState` - see **Data Structures** below for details (it's big and shouldn't be repeated all over this document). Regarding error states: non-existent jobs have the status `does_not_exist`, and when the job state cannot be retrieved from EE2 the status `ee2_error` is used
-  * `outputWidgetInfo` - the parameters to send to output widgets, only available for a completed job
-
 Sample response JSON:
 ```json

```

Comment:
Removing the "json" tag here will fix the formatting issues at the cost of losing colors.